bootstrap/eksctl/README.md

```diff
@@ -163,6 +167,35 @@ kubectl wait --for condition=established --timeout=300s crd/providerconfigs.kube
 kubectl apply -f crossplane/kubernetes-provider-config.yaml
 ```
 
+### Install Crossplane Helm Provider (required for examples that deploy Helm charts)
+```bash
+kubectl create serviceaccount helm-provider -n crossplane-system
+kubectl apply -f crossplane/helm/clusterrolebinding.yaml
+kubectl apply -f crossplane/helm/controller-config.yaml
+kubectl apply -f crossplane/helm/provider.yaml
+# wait for the Helm provider CRD to be ready
+kubectl wait --for condition=established --timeout=300s crd/providerconfigs.helm.crossplane.io
+kubectl apply -f crossplane/helm/provider-config.yaml
+```
+### Deploy ArgoCD in cluster with Crossplane Helm Provider (required for examples that use ArgoCD)
+> Note: The default ArgoCD configuration needs 3 nodes in separate AZs to deploy correctly. By default, eksctl deploys with 2 nodes and no autoscalers.
+
+```bash
+kubectl create namespace argocd
+kubectl apply -f crossplane/argocd/argocd-values-configmap.yaml
+kubectl apply -f crossplane/argocd/argocd.yaml
+```
```

Comment:
We don't need to use the Helm provider to install ArgoCD. I think we can just install straight from the terminal just like the rest of the components.

Reply:
Since the Helm provider is being installed decided to show the use of crossplane to install the release for ArgoCD. I can change to to a helm install from CLI but though thought was nice to show use of crossplane.

Reply:
I agree that it's a nice way to show that you can use the helm provider. I see two issues:

1. Users are likely new to Crossplane. This means trying to debug issues caused by this particular step would involve going through the debugging steps of Crossplane before they can start debugging any helm issues. A hard ask for people new to Crossplane.
2. Consistency. All steps before this do not use Crossplane to bootstrap. I think it's better to keep the steps similar so as not to confuse users.

Reply:
I follow. I will switch out to helm install from cli

Reply:
Changes made. Overall deployment steps in eksctl readme retested and s3-irsa-app in examples/upbound-aws-provider/composite-resources/s3-irsa deployed and sync'd to confirm components are working.